### PR TITLE
F5.1 Header #137 

### PR DESCRIPTION
**Pull request by calhuy**
This addresses issue #137.

Created a basic header using an AppBar, did not include the mesh logo since we are waiting for the import but should be easy to place.

image

### CODE REVIEW

File: meshapp/src/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+const Header = () => {
+
+    return (
+        <AppBar position="static" sx={{ background: "#0B7D66" }}>
+
+            <Toolbar>
+                {/*---Left Side Menu Elements (Menu/Notification Icon)---*/}
+                <Grid container>
+                    <IconButton
+                        sx={{ mr: 2, color: "#0000008F" }}
```
Comment (PattyCakesxD) [inline meshapp/src/components/Header.tsx:24]: The color hex ("#0000008F") seems wrong, what's it for?

File: meshapp/src/components/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                    >
+                        <MenuIcon sx={{ fontSize: "40px" }} />
+                    </IconButton>
+
+                    <IconButton
+                        size="large"
+                        edge="start"
+                        color="inherit"
+                        aria-label="menu"
+                        sx={{ mr: 2, color: "#0000008F" }}
```
Comment (PattyCakesxD) [inline meshapp/src/components/Header.tsx:34]: The color hex ("#0000008F") seems wrong, what's it for?

(Repeat)

**Review by PattyCakesxD — APPROVED** [review]
Use prettier extension to format the file

rest lgtm! 👍

File: meshapp/src/components/Header.tsx (hunk 1 above)
Comment (calhuy) [reply meshapp/src/components/Header.tsx:24]: I followed the figma and the icons had that color. 😄

**Review by Darren52BT — APPROVED** [review]
lgtm

**Pull request merged**